### ISSUE TITLE
Allow text orientation by default on any block that has a typography panel

### ISSUE THREAD

**Issue by beafialho** [WordPress/gutenberg#62693]
Currently, as far as I know, only the `Paragraph` and `Heading` blocks can become vertical text. I suggest, as mentioned in #50319 and specifically in this comment, allowing `text-orientation` by default on any block that has a typography panel. Here's an example of a cool layout we'd be able to achieve below with a vertical Site Title:

![site title](https://github.com/WordPress/gutenberg/assets/65220155/30794c32-d48a-455e-8971-9549915ec16b)

**Comment by MaggieCabrera** [comment]
I think adding this to all blocks with a typography panel might be a bit too much. Blocks like the search block, the comments block or the table of contents may not look ok with this setting. I'll start with the ones that make more sense beyond the heading and the paragraph though

**Comment by beafialho** [comment]
> I think adding this to all blocks with a typography panel might be a bit too much. Blocks like the search block, the comments block or the table of contents may not look ok with this setting. I'll start with the ones that make more sense beyond the heading and the paragraph though

Yes, I understand what you mean and I think the search block probably isn't a good one either, there may be others too. But Site Title, Post Navigation and Buttons for instance sound like some good ones to start.

**Comment by genepine** [comment]
Hi
When I set the typography text to vertical in the WordPress 6.7.1 editing screen, I can't move between characters with the cursor keys.
![Image](https://github.com/user-attachments/assets/908092d8-7bc6-4d20-87f6-e47e4acfee38)
![Image](https://github.com/user-attachments/assets/a012c689-2daa-455d-ab59-20cf3d87bf50)

*If multiple lines of vertical text are lined up, the vertical text will be lined up vertically. (Maybe I didn't think about this enough.)
This requires specifying vertical text for the group block and specifying that the effect should be applied to the paragraphs contained within.

If I leave the text horizontal in the editing screen and specify vertical text in CSS, I can move between characters with the cursor keys.
![Image](https://github.com/user-attachments/assets/b944854c-5e8e-4149-a271-2b85253536bf)
![Image](https://github.com/user-attachments/assets/62efa868-6995-4021-91da-06749b2cdef7)

https://github.com/user-attachments/assets/b7e66883-74e5-49bf-9c4e-49175062f655

**Comment by t-hamano** [comment]
Hi @genepine, thanks for the report.

This issue is for discussing enhancements. For bugs, Could you fsubmit a new issue?

**Comment by genepine** [comment]
@t-hamano 
Thanks guiding me,

I submit a new issue.
https://github.com/WordPress/gutenberg/issues/67394
